app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,45 @@
+<%= render partial: "search", locals: { page_title: "Welcome to Muncher"} %>
+
+<section class="show-page">
+
+  <h2>
+     <%= link_to @recipe.name, "#{@recipe.url}", target: :_blank %>
+  </h2>
+
+  <section class="recipe-info">
+    <%= image_tag(@recipe.image, alt: "food-image", class: "food-pic") %>
+    <section class="list-info">
+      <ul>
+        <h3 class="blue-mc">
+          Ingridients:
```

Comment:
ingredients, careful with spelling

Reply:
OOOOOOps! Thank you Chris!

Reply:
I have ingredients spelled wrong everywhere in my project, that's why it did not break.

Reply:
Corrected it.